src/routes/AppRoutes.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+
+import Home from '../components/Home/Home.jsx';
+import Example from '../components/Example/Example.jsx';
+
+export default function AppRoutes() {
+  return (
+    <Routes>
+      <Route path="/" element={<Home />} />
```

Comment:
En vez de usar element podrías usar component y pasar directo el componente:
```js
<Route path="/" component={Home} />
```
Lo mismo para el otro

Reply:
Se cambio la implementación en [React Router v6](https://reactrouter.com/docs/en/v6/api#routes-and-route), parece ser que `component` como propiedad ya no es soportado.
Encontré esta misma pregunta en [Stack Overflow](https://stackoverflow.com/questions/69877208/property-component-does-not-exist-on-type)

Reply:
Bien si, no me percaté de la versión, acostumbrado a ver componente. Barbaro!